css/css-flexbox-exercise-1.css

```diff
@@ -0,0 +1,119 @@
+body {
+    margin: 0% 5%;
+    
+}
+
+h1 {
+    font-size: 24px;
+    font-weight: 600;
+    padding: 15px 15px;
+    margin-top: 10px;
+    border-top: 1px solid #000;
+    border-bottom: 3px solid #48B99D;
+}
+
+.shopping_list {
+    margin-top: 5px;
+    display: flex; 
+    justify-content: space-between;
+    border-top: 3px solid #48B99D;
+    column-gap: 5px; 
+    
+    
+}
+
+.box {
+    display: flex;
+    flex-direction: column; 
+    align-items: flex-start;
+    justify-content: center;
+    margin-top:10px;
+    border: 1px solid #000;
+    border-radius: 5px;
+    flex-basis: 100%;
+}
+
+.background-img-1 {
+    display: flex;
+    align-items: center;
+    background-image: url("../img/warzywa.png");
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 150px;
+}
+
+.background-img-2 {
+    display: flex;
+    align-items: center;
+    background-image: url("../img/owoce.jpg");
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 150px;
+}
+
+.background-img-3 {
+    display: flex;
+    align-items: center;
+    background-image: url("../img/nabial.jpg");
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 150px;
+}
+h2 {
+    /* width: 50px; */
+    background-color: #fff;
+    opacity: 0.7;
+    text-transform: uppercase;
+    margin-left: 20px;
+    padding:10px;
+    width: 150px;
+    font-size: 18px;
+    font-weight: 600;
+}
+
+.product-list {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    padding-left: 30px;
+    height: 150px;
+    
+        
+}
+
+.ui__ul {
+    list-style:inside;
+}
+
+li:hover {
+    background-color: #48B99D; 
+    color:#fff;
+}
+
+
+.sub-lista {
+    margin-left:40px;
+    list-style:inside;
+}
+
+.checkbox {
+    display: flex;
+    justify-content: flex-end;
+    background-color: #48B99D;
+    align-items: center;
+    width: 100%;
+    text-align: right;
+    height: 50px;
+}
+
+label {
+    padding: 10px;
+}
+
+footer>p {
+    font-size: 18px;
+    font-weight: 600;
+    margin-top: 40px;
+    margin-left:10px;
+
```

Comment:
a tak pozatym to spoko te cssy, proste selektory, nazwy też fajne :) jak na poczatki to bardzo fajnie :+1: